test/inductor/test_torchinductor.py

```diff
@@ -4799,6 +4799,50 @@ def fn(a):
         self.assertTrue((d >= 0).all())
         self.assertTrue((d < 1).all())
 
+    def test_functionalize_rng_wrappers(self):
+        # Ideally, we would like to use torch.compile for these operators. But
+        # currently the plan is to introduce these operators at the partitioner
+        # level, obviating the need to support them fully through the
+        # torch.compile stack. To ensure that we have good enough debugging with
+        # minifiers, we have ensure that they work with make_fx. This test uses
+        # make_fx to do the testing. In future, we can move on torch.compile.
+        def fn():
+            rng_state1, a1 = torch.ops.run_and_save_rng_state(
```

Comment:
The ones in aten/prims do not accept a torch.ops.OpOverload as an operand. It was unclear what the schema for such ops should look like. So, higher order op seemed a better fit to me. What do you think?

Reply:
This is a pre-existing condition. All of the HigherOrderOps are in the global namespace (which is problematic for other things), we should fix it at some point

Reply:
@anijain2305 My plan is to stop exposing HigherOrderOperators under torch.ops.*. The main reasons are:
1) redundancy. We're going to have e.g. torch.cond, having it also as torch.ops.cond is redundant. Comparing this to the rest of torch.ops, having aten operators like torch.ops.aten.sin is reasonable because there is no other way to access them from Python.
2) Being in a global namespace leads to problems

The implication is that we should expose `run_and_save_rng_state` as e.g. torch._prims.rng_prims.run_and_save_rng_state and access them from there directly instead of under torch.ops

Reply:
Yes, that makes sense. Let me change the code to access the ops.